run/deployment-previews/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use the Cloud SDK docker container, which includes Python 3.7
+# https://github.com/GoogleCloudPlatform/cloud-sdk-docker
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:slim
```

Comment:
For consistency and teach purposes, can you add more comments to this file. Inspiration https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/Dockerfile and go/dockerfile-sample-practices